generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -247,12 +255,13 @@ Scenario: Running a 'shorterThan' request as part of a non-contradicting anyOf c
          { "field": "foo", "is": "shorterThan", "value": 3 }
        ]}
        """
-       And foo is containing regex /[%]{1}/
-     Then the following data should be included in what is generated:
+       And foo is matching regex /[%]{1,10}/
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
        | foo  |
        | null |
-       | "%1" |
        | "%"  |
+       | "%%" |
        And the following data should not be included in what is generated:
```

Comment:
After changing the above to define exactly what should be generated you could remove this`And the following data should not be included in what is generated:` section

Reply:
There are other places in the file where I think this can also be done, but changes haven't been made to this area so I can't comment.

Reply:
agreed, removed